### PR TITLE
OpenBSD: update packages list

### PR DESCRIPTION
**Pull request by osokin**

### CODE REVIEW

**Review by danielledeleo — APPROVED** [review]

File: source/installation.rst [hunk 1]
```diff
@@ -2291,6 +2291,7 @@ Community Repositories
            # pkg_add unit-php80
```
Comment (danielledeleo) [inline source/installation.rst:2291]: Is php8.0 still supported on OpenBSD?

Comment (ac000) [reply source/installation.rst:2291]: It still seems to be in [ports](https://github.com/openbsd/ports/tree/master/lang/php) and the versions there do seem to match with what's listed in this file, but then I have little knowledge of the workings of the *BSD's.

**Comment by danielledeleo** [comment]
@osokin This is good to merge. Go ahead and click the button.

**Comment by osokin** [comment]
> @osokin This (and the others) are good to merge. Go ahead and click the button.

@danielledeleo Seems like I don't see any buttons here, where is that?

**Pull request merged**